crates/unified-bridge/Cargo.toml

```diff
@@ -8,22 +8,22 @@ license.workspace = true
 zkvm = []
 testutils = ["dep:arbitrary", "agglayer-primitives/testutils"]
 
+[lints]
+workspace = true
+
 [dependencies]
 agglayer-primitives.workspace = true
 agglayer-tries.workspace = true
 
 arbitrary = { workspace = true, optional = true }
-hex-literal = "0.4"
-serde = { version = "1", features = ["derive"] }
-serde_with = { version = "3" }
-sha2 = { git = "https://github.com/sp1-patches/RustCrypto-hashes.git", package = "sha2", tag = "patch-sha2-0.10.8-sp1-4.0.0" }
```

Comment:
Just to make sure, the proof programs still pick up on the patched crates due to the overrides, correct? Would be nice to verify this by recompiling pp and re-running the cycle tracker tests (if the elf happens to be identical, even better).

Reply:
> Just to make sure, the proof programs still pick up on the patched crates due to the overrides, correct? Would be nice to verify this by recompiling pp and re-running the cycle tracker tests (if the elf happens to be identical, even better).

Ah, now I see the top-level comment, feel free to ignore.